### PR TITLE
Align morty directory structure to upstream

### PR DESCRIPTION
**Pull request by thenor**
Move variant and configurations to a conf directory.
Add BSP layers for Intel and Rasberry Pi.
Remove mandatory template service from pelux bbclass.
Add glib-2.0-native depends.

### CODE REVIEW

File: conf/variant/rpi-qtauto/conf-notes.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Available PELUX images:
+<<<<<<< HEAD:meta-rpi-extras/conf/conf-notes.txt
+* core-image-pelux
+=======
+* core-image-pelux-minimal
+* core-image-pelux-qtauto-neptune
+>>>>>>> 023c0df... Move variants and configurations to conf directory:conf/variant/rpi-qtauto/conf-notes.txt
```
Comment (tobsan) [inline conf/variant/rpi-qtauto/conf-notes.txt]: This is a conflict.

File: meta-rpi-extras/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+META-PELUX-BSP-RPI
```
Comment (tobsan) [inline meta-rpi-extras/README.md]: I think there is a commit in pyro that changes the name here.

File: conf/variant/rpi-qtauto/conf-notes.txt (hunk 1 above)
Comment (thenor) [reply conf/variant/rpi-qtauto/conf-notes.txt]: On it.

File: meta-rpi-extras/README.md (hunk 2 above)
Comment (thenor) [reply meta-rpi-extras/README.md]: Done.

**Pull request merged**